pkg/services/validator/validator_service.go

```diff
@@ -240,6 +241,13 @@ func handlePlugins(vc *components.ValidatorConfig, tc *cfg.TaskConfig, kClient k
 				return err
 			}
 		}
+
+		log.InfoCLI("Select the Azure cloud environment to connect to.")
```

Comment:
Please move this into `readAzurePlugin` for consistency. We don't have any plugin configuration prompts in this function, just calls to other prompt functions.